tests/envs/test_cover.py

```diff
@@ -11,7 +11,7 @@
 
 def test_cover():
     """Tests for CoverEnv class."""
-    utils.update_config({"env": "cover"})
+    utils.update_config({"env": "cover", "cover_initial_holding_prob": 0.5})
```

Comment:
This is a sort of weird test. It would be better to avoid randomness in the tests, instead of doing this, just add a new test that sets this prob to 1.0, and check that all the train/tests have some object being held...
`assert sum(state.get(obj, "held") for obj in state if obj.type.name == "block") == 1`
and you can add this same assert but with `0` at the end for the existing test.